include/nncase/codegen/module_builder.h

```diff
@@ -93,6 +93,7 @@ class NNCASE_API module_builder
     section *find_section(std::string_view section_name);
     void merge_to_rdata_section(std::string_view from);
     function_call_id function_id(ir::graph *graph);
+    uint32_t get_current_entry_point();
```

Comment:
Getter's return type should be same with setter's argument type (std::std::streampos).